dashboard/src/main/resources/webapp/static/lib/angular-topology-editor/canvas.js

```diff
@@ -101,7 +101,8 @@ Graph.Node.popovercontent = function(i) {
 };
 
 var Canvas = (function() {
-
+    var LOG = Log.getLogger("Canvas").setLevel(Log.INFO);
```

Comment:
log was defined as a global variable, and I had two choices because two log libraries did exist. I took this one because it felt more simple to me.